LongWarOfTheChosen/Config/XComLW_Overhaul.ini

```
@@ -1267,6 +1267,9 @@ AutopsyAdventPsiWitchIntelCost=0
 +GTSTable=(GTSProjectTemplateName="SquadSizeIIUnlock",			SupplyCost=75,	RankRequired=8,		HideIfInsufficientRank=true,	UniqueClass="Ranger");
 +GTSTable=(GTSProjectTemplateName="FNGUnlock",					SupplyCost=375,	RankRequired=9,		HideIfInsufficientRank=true,	UniqueClass="Ranger");
 
+; This option will let you disable and hide GTS unlocks from other mods.
+;+GTSUnlocksToRemove=""
```

I would expand this to a full example, including the name of a real GTS unlock. For example:
```suggestion
;+GTSUnlocksToRemove="SquadSizeIUnlock"
```
It just helps showing what the name of a GTS unlock typically looks like.

---

Coolsies, sounds good.